javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,71 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+const indexOfNameToRemove= names.indexOf(nameToRemove);
+names.splice(indexOfNameToRemove,1);
+console.log(names);
+
+
+
+//when will we be there?
+//https://www.webdeveloper.com/d/273645-how-to-convert-decimal-time-value-to-hhmm/2
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+}; 
+function getTime(travelInfo) {
+    const time=travelInfo.destinationDistance/travelInfo.speed;
+    const hours=parseInt(Number(time));
+    const minutes=Math.round((Number(time)-hours) * 60);
+    return hours+' hours and '+minutes+' minutes';
+}
```

Comment:
Consistent formatting is important, Google for "vs code format document" to learn how to format your document consistently.

Reply:
Did formatting for all the js pages